src/components/atomic/Tag.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { HTMLAttributes } from "react";
+import classNames from "classnames";
+
+type TagProps = HTMLAttributes<HTMLDivElement>;
```

Comment:
We can probably just directly put HTMLAttributes<HTMLDivElement> as the type- not sure what other devs think about this since we do have the same format in Input and Card despite having no extra props

Reply:
Changed!